docs/scos/dev/feature-integration-guides/202307.0/glue-api/dynamic-data-api/dynamic-data-api-integration.md

```diff
@@ -356,7 +453,41 @@ class GlueBackendApiApplicationDependencyProvider extends SprykerGlueBackendApiA
     }	    
 }
 ```
-</details>
+
+
+**src/Pyz/Zed/Installer/InstallerDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Installer;
+
+use Spryker\Zed\DynamicEntity\Communication\Plugin\Installer\DynamicEntityInstallerPlugin;
+use Spryker\Zed\Installer\InstallerDependencyProvider as SprykerInstallerDependencyProvider;
+
+class InstallerDependencyProvider extends SprykerInstallerDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\Installer\Dependency\Plugin\InstallerPluginInterface|\Spryker\Zed\InstallerExtension\Dependency\Plugin\InstallerPluginInterface>
+     */
+    public function getInstallerPlugins(): array
+    {	    
+        return [
+            new DynamicEntityInstallerPlugin(),
+        ];
+    }	    
+}
+```
+
+{% info_block warningBox %}
+
+When `DynamicEntityInstallerPlugin` is enabled, run the following command:
```

Comment:
Instead of "run the following command", focus on what the command does (e.g.  
"When `DynamicEntityInstallerPlugin` is enabled, initiate a database"